hyper/hyper/restore_test.go

```diff
@@ -0,0 +1,83 @@
+package hyper_test
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRestore(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareTest func(h *hyper.Hyper) string
+		ExpectedErr string
+	}{
+		"restore the desktop correctly": {
+			PrepareTest: func(h *hyper.Hyper) string {
+				desktop, err := h.Start(hyper.TestMinDesktopXML(t), &hyper.StartOptions{})
+				require.NoError(err)
+
+				dir, err := ioutil.TempDir("", "dumps")
+				if err != nil {
+					log.Fatal(err)
+				}
```

Comment:
D.R.Y! Also, it should have a name like `isard-test-restore` or something like this

Reply:
ok